README.md

```diff
@@ -207,6 +207,19 @@ Specifies verbosity. Default is `true`.
 In verbose mode, the `zx` prints all executed commands alongside with their outputs.
 This is the same as using `set -x` in Bash.
 
+### `$.env`
+
+It's an alias for `process.env`.
+
+### `$.argv`
+
+It is an array containing arguments received by the current script.
```

Comment:
- This should note that `$.argv` drops the first 2 args of `process.argv`.
- Descriptions like these read better if you drop the "it's".

```suggestion
An alias for `process.env`.

### `$.argv`

An array containing the arguments received by the current script.
Excludes the execution path and filename.
```

Reply:
`Excludes the execution path and filename.` may bring ambiguity to users. I think I need to add more detailed explanation. I will add it. Thank you!